backend/modbus/modbus_client.py

```diff
@@ -0,0 +1,102 @@
+from pymodbus.client.sync import ModbusTcpClient as ModbusClient
+from pymodbus.payload import BinaryPayloadDecoder
+from pymodbus.constants import Endian
+import requests
+from decouple import config
+from datetime import datetime
+
+
+'''
+get_token(): Function that generates access and refresh tokens for authentication and access to API data
+
+returns:
+    access_token: Has an expiration time of 5 minutes.
+    refresh_token: Token used for generating a new access_token after expiration. refresh_token has expiration time of 24hrs.
+'''
+def get_token():
```

Comment:
You have to put the docstrings inside the actual function, otherwise I think python doesn't associate it to the function.